src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -239,8 +239,8 @@ def test_storage_azcopy_blob_url(self, resource_group, first_account, second_acc
         content_type = "application/json"
         self.cmd('storage copy -s "{}" -d "{}" --content-type {}'.format(
             os.path.join(test_dir, 'readme'), first_container_url, content_type))
-        self.cmd('storage blob list -c {} --account-name {}'
```

Comment:
Since you bump the version of azcopy, I believe you have run this test lively. Is that correct?

Reply:
yes